bindings/python/rgbmatrix/core.pyx

```diff
@@ -72,6 +78,17 @@ cdef class FrameCanvas(Canvas):
     def SetPixel(self, int x, int y, uint8_t red, uint8_t green, uint8_t blue):
         (<cppinc.FrameCanvas*>self.__getCanvas()).SetPixel(x, y, red, green, blue)
 
+    def Serialize(self):
+        cdef const char* data
+        cdef size_t length
+        (<cppinc.FrameCanvas*>self.__getCanvas()).Serialize(&data, &length)
```

Comment:
I'm afraid that this won't work for your purpose. Serialize just returns pointer to internal frame representation (one instance per FrameCanvas), it does not copy it.

One option is to implement buffer protocol for `FrameCanvas` and then use python for memory management. I'll post PR soon